### PR TITLE
Connect Redux App with Books API

### PR DESCRIPTION
**Pull request by RafaelEchart**
## Milestone review:

- Created a new game in the [Books API](https://www.notion.so/Bookstore-API-51ea269061f849118c65c0a53e88a739)
- Added the function for adding books
- Added the function for removing books
- When the page loads a get request is done to the API to display the available books 

## INSTRUCTIONS: 
-npm install
-npm run start

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -13,12 +15,27 @@ export const removeBook = (payload) => ({
...
     case ADD_BOOK:
+      newBook(action.payload);
       return [...state, action.payload];
 
     case REMOVE_BOOK:
+
+      deleteBook(action.payload);
       return state.filter((book) => book.id !== action.payload);
 
     default:
```
Comment (ArthurGC) [inline src/redux/books/books.js]: - [ ] There are some console errors that appear in your project. The video explains those so please watch it. 😉 

image

image


https://user-images.githubusercontent.com/79876959/145360677-f26acc9e-deeb-491e-9c89-1c3003c66915.mp4

**Review by ArthurGC — CHANGES_REQUESTED** [review]
# Required Changes ♻️ :warning: 
Hi  @RafaelEchart 

Amazing job!
There are some issues that you still need to work on to go to the next project but you are almost there! 



## Good Points :+1: :
You used proper GitHub Flow :heavy_check_mark:
Your linter checks are all passing :heavy_check_mark:
Your PR messages are descriptive :heavy_check_mark:

## Aspects to improve ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/redux/books/books.js (hunk 1 above)
Comment (RafaelEchart) [reply src/redux/books/books.js]: Hello @ArthurGC, this is the best CR I have ever got, I really wanted to thank you for the time you took in reviewing my code and making this video, it was really helpful!!

**Review by AshakaE — APPROVED** [review]
### Hi @  🙋‍♀️, 

# STATUS APPROVED ✔

Thanks for making the requested change 😄 
   
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**